almost_tcp/packet_fixtures_test.py

```diff
@@ -0,0 +1,56 @@
+from amaranth.lib.fifo import SyncFIFOBuffered
+from amaranth.sim import Simulator
+from hypothesis import given
+from packet_fixtures import MultiPacketSender, PacketCollector
+from message_host import Packet
+from typing import List
+import sys
+
+
+@given(packets=...)
+def test_stream_send_structured_receive(packets: List[Packet]):
+    """
+    Test a round-trip between MultiPacketSender (Packet -> stream) and
+    PacketCollector (stream->Packet), via a FIFO.
+    """
+
+    # All we need is a buffer for this to work.
+    # Depth chosen arbitrarily.
+    dut = SyncFIFOBuffered(width=8, depth=2)
+    sender = MultiPacketSender(random_delay=True, stream=dut.w_stream)
+    # Turning on random backpressure here causes the tests to take A Long Time.
```

Comment:
That's kind of surprising. I would have expected it to roughly double the amount of time, but not dramatically increase it.

(no action required)

Reply:
It might be because of Hypothesis? Hypothesis interposes the PRNG, so its tests cases are deterministic & reproducible; I don't know if it goes further and produces non-pseudorandom input for the RNG (e.g. keeps us at "invalid indefinitely").